### PR TITLE
Refresh library landing page (replacement)

### PR DESCRIPTION
**Pull request by petele**

### CODE REVIEW

File: src/content/en/tools/service-worker-libraries/index.md [hunk 1]
```diff
@@ -3,30 +3,64 @@ book_path: /web/tools/_book.yaml
...
 
-## What are the Service Worker Libraries?
+<div class="attempt-right">
+  <figure>
+    <img src="../images/tools-landing-page.gif" max-width="50%">
+    <figcaption>Overview of Service Worker Libraries</figcaption>
+  </figure>
+</div>
+
+Use our [service worker](https://github.com/slightlyoff/ServiceWorker/blob/master/explainer.md)
```
Comment (petele) [inline src/content/en/tools/service-worker-libraries/index.md]: This needs to point to our Service Worker intro

File: src/content/en/tools/service-worker-libraries/index.md [hunk 2]
```diff
@@ -3,30 +3,64 @@ book_path: /web/tools/_book.yaml
...
 {# wf_published_on: 2015-01-01 #}
-{# wf_updated_on: 2016-09-12 #}
+{# wf_updated_on: 2016-11-03 #}
 
 # Service Worker Libraries {: .page-title }
 
-## What are the Service Worker Libraries?
+<div class="attempt-right">
+  <figure>
+    <img src="../images/tools-landing-page.gif" max-width="50%">
```
Comment (petele) [inline src/content/en/tools/service-worker-libraries/index.md]: Do not use max-width on this, it'll screw up on mobile.

Comment (petele) [inline src/content/en/tools/service-worker-libraries/index.md]: Please use the absolute path here `/web/tools/images/...`

File: src/content/en/tools/service-worker-libraries/index.md [hunk 3]
```diff
@@ -3,30 +3,64 @@ book_path: /web/tools/_book.yaml
...
+    <figcaption>Overview of Service Worker Libraries</figcaption>
+  </figure>
+</div>
+
+Use our [service worker](https://github.com/slightlyoff/ServiceWorker/blob/master/explainer.md)
+libraries to simplify your development by eliminating service worker boilerplate
+code.
+
+**sw-precache&mdash;**Integrates with your build process to generate a service
```
Comment (petele) [inline src/content/en/tools/service-worker-libraries/index.md]: these should be links to the right section below

File: src/content/en/tools/service-worker-libraries/index.md [hunk 4]
```diff
@@ -3,30 +3,64 @@ book_path: /web/tools/_book.yaml
...
+You can address all of these drawbacks using our service worker libraries.
 
 
 ## Service Worker Precache 
 
 [Service Worker Precache](https://github.com/GoogleChrome/sw-precache/) (`sw-precache`) is a
 module for generating a service worker that
 precaches resources. The module can be used in JavaScript-based build scripts,
-like those written with [`gulp`](http://gulpjs.com/){: .external }, and it also provides a
+like those written with [`gulp`](http://gulpjs.com/), and it also provides a
```
Comment (petele) [inline src/content/en/tools/service-worker-libraries/index.md]: Nit - Please update gulpjs.com to https:// or //gulpjs.com

File: src/content/en/tools/service-worker-libraries/index.md (hunk 2 above)
Comment (jpmedley) [inline src/content/en/tools/service-worker-libraries/index.md]: Did width="80%". 100 covers the page navigation. 80 has appropriate behavior on a variety of device and screen sizes.

Comment (jpmedley) [inline src/content/en/tools/service-worker-libraries/index.md]: Done.

File: src/content/en/tools/service-worker-libraries/index.md (hunk 1 above)
Comment (jpmedley) [inline src/content/en/tools/service-worker-libraries/index.md]: Done.

File: src/content/en/tools/service-worker-libraries/index.md (hunk 4 above)
Comment (jpmedley) [inline src/content/en/tools/service-worker-libraries/index.md]: Done.

**Comment by jpmedley** [comment]
PTAL

File: src/content/en/tools/service-worker-libraries/index.md [hunk 5]
```diff
@@ -3,30 +3,64 @@ book_path: /web/tools/_book.yaml
...
 {# wf_published_on: 2015-01-01 #}
-{# wf_updated_on: 2016-09-12 #}
+{# wf_updated_on: 2016-11-03 #}
 
 # Service Worker Libraries {: .page-title }
 
-## What are the Service Worker Libraries?
+<div class="attempt-right">
+  <figure>
+    <img src="/web/tools/images/tools-landing-page.gif" width="80%">
```
Comment (petele) [inline src/content/en/tools/service-worker-libraries/index.md]: Remove all widths! `attempt-right` is designed to be responsive, if you add a size, you're breaking the point of that.

File: src/content/en/tools/service-worker-libraries/index.md [hunk 6]
```diff
@@ -3,30 +3,64 @@ book_path: /web/tools/_book.yaml
...
+    <figcaption>Overview of Service Worker Libraries</figcaption>
+  </figure>
+</div>
+
+Use our [service worker](/web/fundamentals/getting-started/primers/service-workers)
+libraries to simplify your development by eliminating service worker boilerplate
+code.
+
+**sw-precache&mdash;**Integrates with your build process to generate a service
```
Comment (petele) [inline src/content/en/tools/service-worker-libraries/index.md]: links?

File: src/content/en/tools/service-worker-libraries/index.md (hunk 5 above)
Comment (jpmedley) [inline src/content/en/tools/service-worker-libraries/index.md]: I tested for that. It works fine.

File: src/content/en/tools/service-worker-libraries/index.md (hunk 6 above)
Comment (jpmedley) [inline src/content/en/tools/service-worker-libraries/index.md]: No. They just duplicate what's in the page contents.

File: src/content/en/tools/service-worker-libraries/index.md (hunk 5 above)
Comment (petele) [inline src/content/en/tools/service-worker-libraries/index.md]: Please remove it, we should not be using it.

**Comment by jpmedley** [comment]
PTAL.

**Pull request merged**